src/main/resources/application.properties

```
@@ -0,0 +1,16 @@
+spring.application.name=SpringExample
+
+spring.datasource.url=jdbc:mysql://localhost:3306/contact
+spring.datasource.username=root
+spring.datasource.password=admin
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+
+#management.endpoints.web.exposure.include = beans,mapping,session
+management.endpoints.web.exposure.include =*
```

```suggestion
management.endpoints.web.exposure.include =health,metrics
```
you are exposing all management endpoints which can be a security risks.